TOCSharp/TOCClient.cs

```diff
@@ -18,6 +18,8 @@ public partial class TOCClient
         private FLAPConnection? connection;
         private readonly TOCClientSettings settings;
 
+        private bool isKeepAliveLoopRunning = false;
```

Comment:
Change variable name to "keepAliveLoopRunning", no need for "is" considering the variable is a boolean

Reply:
Addressed in latest commit.